src/App.tsx

```diff
@@ -1,18 +1,41 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [itemsLoad] = useState(items);
+  const [itemsPerPage, setItemsPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState<number>(1);
+
+  const handleChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setItemsPerPage(+event.target.value);
+    setCurrentPage(1);
+  };
+
+  const indexOflastItem = currentPage * itemsPerPage;
+  const indexOfFirstItem = indexOflastItem - itemsPerPage;
+  const currentItem = itemsLoad.slice(indexOfFirstItem, indexOflastItem);
+
+  const lastItem = Math.min(itemsPerPage * currentPage, itemsLoad.length);
+  const firstItem = itemsPerPage * (currentPage - 1) + 1;
+
+  const onPageChange = (page: number) => {
```

Comment:
```suggestion
  const handlePageChange = (page: number) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364